AUTHORS

```diff
@@ -22,3 +22,4 @@ Ruslan Kovalov <ruslan.kovalyov@gmail.com>
 Venafi, Inc.
 Vladimir Rutsky <vladimir@rutsky.org>
 Ximin Luo <infinity0@gmx.com>
+Laël Cellier
```

Comment:
As per the instructions above, you should have your email address, and keep this list sorted. Also, if I'm deciphering these funny characters on GitHub properly, I think you might be missing a newline?